atd-vze/src/views/VZDashboard/VZDashboard.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Col, Row } from "reactstrap";
+import { withApollo } from "react-apollo";
+import { useQuery } from "@apollo/react-hooks";
+import Widget02 from "../Widgets/Widget02";
+
+import { GET_CRASH } from "../../queries/dashboard";
+
+function VZDashboard() {
+  const year = new Date().getFullYear();
+  const yearStart = `${year}-01-01`;
+  const yearEnd = `${year}-12-31`;
+  const { loading, error, data } = useQuery(GET_CRASH, {
+    variables: { yearStart, yearEnd },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  const yearsOfLifeLostYTD =
+    data.atd_txdot_primaryperson_aggregate.aggregate.sum.years_of_life_lost +
+    data.atd_txdot_person_aggregate.aggregate.sum.years_of_life_lost;
+  const fatalitiesYTD =
+    data.atd_txdot_crashes_aggregate.aggregate.sum.death_cnt;
+  const seriousInjuriesYTD =
+    data.atd_txdot_crashes_aggregate.aggregate.sum.sus_serious_injry_cnt;
+  const crashesYTD = data.atd_txdot_crashes_aggregate.aggregate.count;
```

Comment:
you're using a really long repeated string here. Let's use object deconstruction to make this look pretty.

```js
const { 
  years_of_life_lost: yearsOfLifeLostPrimaryPerson
} = data.atd_txdot_primaryperson_aggregate.aggregate.sum

const { 
  years_of_life_lost: yearsOfLifeLostPerson
} = data.atd_txdot_person_aggregate.aggregate.sum

const {
  death_cnt: deathCount,
  sus_serious_injry_cnt: seriousInjuryCount
} = data.atd_txdot_crashes_aggregate.aggregate.sum

const {
  count: crashesCount
} = data.atd_txdot_crashes_aggregate.aggregate
```

more verbose of a set up but maybe nicer to read?

Reply:
or maybe not, your call...

Reply:
I think we should also format these counts to use commas for thousands separators to make it more human-readable.

https://stackoverflow.com/questions/2901102/how-to-print-a-number-with-commas-as-thousands-separators-in-javascript

Reply:
I went ahead with the object deconstruction you suggested.